.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy to PyPI
+
+on:
+  push: 
+    tags:
+      - v*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
+      - name: Install Setuptools
+        run: pip install setuptools
+      - name: Build Package
+        run: python setup.py bdist
```

Comment:
This doesn't seem right as it *only* seems to generate a bdist (binary distribution) and not also an sdist (source distribution).  I'm pretty sure we should always be releasing an sdist.

When I try this locally I end up with only cdn-definitions-2.3.0.linux-x86_64.tar.gz, which is not even a wheel (the preferred binary distribution format I guess).  I believe releases made this way wouldn't be installable on non-x86_64 arches.

I'm not certain what's the best things to be uploading, but I'm pretty sure bdist alone isn't it.  If in doubt I'd suggest first trying 'sdist bdist_wheel', and if bdist_wheel hits any problems then going with just 'sdist'.

Reply:
Thanks! I have changed the build to build both wheel and source packages. 

(The results can be downloaded in test.pypi.org: https://test.pypi.org/project/cdn-definitions/#files )